tools/xtce_generation/telemetry_generator.py

```diff
@@ -110,12 +110,13 @@ def create_parameter_types(self, parameter_type_set, unique_lengths):
                     parameter_type_set, "xtce:IntegerParameterType"
                 )
                 parameter_type.attrib["name"] = parameter_type_ref_name
-                parameter_type.attrib["signed"] = "false"
+                parameter_type.attrib["signed"] = false
```

Comment:
```suggestion
                parameter_type.attrib["signed"] = False
```
Is false defined above to be a string? If so it is confusing. Or do you want this to be the boolean and need the caps?

Reply:
It should be a string, I'll update that.